src/callstack.cr

```diff
@@ -17,6 +17,10 @@ require "callstack/lib_unwind"
   require "debug/dwarf"
 {% end %}
 
+# Returns the execution stack (a backtrace) of the current
```

Comment:
As per @ysbaddaden's comment, how about we clarify that this is a human readable execution stack and add a note that it can be used while debugging. Then I think it's ready to merge.